examples/basic/cluster.tf

```diff
@@ -0,0 +1,44 @@
+module "cluster" {
+  source  = "vistaprint/simple-eks/aws"
+  version = "0.3.3"
+
+  cluster_name       = "simple-eks-integration-test-for-eks-node-group"
+  cluster_version    = "1.20"
+  vpc_name           = var.vpc_name
+  log_group_name     = "a-test-log-group-name"
+
+  region  = var.aws_region
+  profile = var.aws_profile
+}
+
+module "spot_node_group" {
```

Comment:
I would do the example with an on-demand node (like t3a.small/medium or something like that). It simplifies the test and also reduces the likelihood of the test failing if no spot instance can be found.